<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>何雨航</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
   </font>
  </property>
  <property name="cursor">
   <cursorShape>CrossCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="cursor">
    <cursorShape>CrossCursor</cursorShape>
   </property>
   <layout class="QGridLayout" name="gridLayout_7">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="cursor">
       <cursorShape>CrossCursor</cursorShape>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_overview">
       <property name="cursor">
        <cursorShape>CrossCursor</cursorShape>
       </property>
       <attribute name="title">
        <string>总览</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QGroupBox" name="group_accountName">
          <property name="title">
           <string>账户名</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="1">
            <widget class="QLabel" name="label_accountName">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QGroupBox" name="group_total">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>总资产</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="1">
            <widget class="QLabel" name="label_cashTypeIcon">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>￥</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="label_totalAmount">
             <property name="text">
              <string>233</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="2">
         <spacer name="horizontalSpacer_overview">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0">
         <spacer name="verticalSpacer_overview">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_stock">
       <property name="cursor">
        <cursorShape>CrossCursor</cursorShape>
       </property>
       <attribute name="title">
        <string>股票</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QTabWidget" name="tabWidget_stock">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab_ShanghaiStock">
           <attribute name="title">
            <string>上海</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="1">
             <widget class="QLabel" name="label_sh">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <spacer name="horizontalSpacer_shanghai">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="2" column="1">
             <widget class="QPushButton" name="pushButton_sh">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>买入</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0" rowspan="3">
             <widget class="QListWidget" name="listWidget_sh">
              <property name="sortingEnabled">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_ShenzhengStock">
           <attribute name="title">
            <string>深圳</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="0" column="1">
             <widget class="QLabel" name="label_sz">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <spacer name="horizontalSpacer_shenzheng">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>354</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="2" column="1">
             <widget class="QPushButton" name="pushButton_sz">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>买入</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0" rowspan="3">
             <widget class="QListWidget" name="listWidget_sz">
              <property name="sortingEnabled">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_own">
           <attribute name="title">
            <string>持有</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_8">
            <item row="0" column="1">
             <widget class="QLabel" name="label_own">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>TextLabel</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <spacer name="horizontalSpacer_own">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="2" column="1">
             <widget class="QPushButton" name="pushButton_own">
              <property name="text">
               <string>卖出</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0" rowspan="3">
             <widget class="QListView" name="listView_own"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <property name="cursor">
        <cursorShape>CrossCursor</cursorShape>
       </property>
       <attribute name="title">
        <string>基金</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_4">
       <property name="cursor">
        <cursorShape>CrossCursor</cursorShape>
       </property>
       <attribute name="title">
        <string>期货</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_5">
       <property name="cursor">
        <cursorShape>CrossCursor</cursorShape>
       </property>
       <attribute name="title">
        <string>外汇</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>CrossCursor</cursorShape>
   </property>
   <widget class="QMenu" name="menu_game">
    <property name="cursor">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="title">
     <string>游戏</string>
    </property>
    <addaction name="action_start"/>
    <addaction name="separator"/>
    <addaction name="action_logout"/>
    <addaction name="separator"/>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menu_edit">
    <property name="cursor">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="action_refresh"/>
    <addaction name="separator"/>
    <addaction name="action_settings"/>
   </widget>
   <widget class="QMenu" name="menu_view">
    <property name="cursor">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="title">
     <string>视图</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_tool">
    <property name="cursor">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="title">
     <string>工具</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_help">
    <property name="cursor">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="action_help"/>
    <addaction name="separator"/>
    <addaction name="action_tutorial"/>
    <addaction name="separator"/>
    <addaction name="action_update"/>
    <addaction name="separator"/>
    <addaction name="action_about"/>
   </widget>
   <addaction name="menu_game"/>
   <addaction name="menu_edit"/>
   <addaction name="menu_view"/>
   <addaction name="menu_tool"/>
   <addaction name="menu_help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="cursor">
    <cursorShape>CrossCursor</cursorShape>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
  </widget>
  <action name="action_start">
   <property name="text">
    <string>开始新游戏</string>
   </property>
  </action>
  <action name="action_logout">
   <property name="text">
    <string>注销账号</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="text">
    <string>退出游戏</string>
   </property>
  </action>
  <action name="action_refresh">
   <property name="text">
    <string>刷新</string>
   </property>
  </action>
  <action name="action_settings">
   <property name="text">
    <string>游戏设置</string>
   </property>
  </action>
  <action name="action_help">
   <property name="text">
    <string>帮助文档</string>
   </property>
  </action>
  <action name="action_tutorial">
   <property name="text">
    <string>新手教程</string>
   </property>
  </action>
  <action name="action_update">
   <property name="text">
    <string>检查更新</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
